backend/README.md

```diff
@@ -1,6 +1,11 @@
 This directory contains code for the components that comprise the Kubeflow
 Pipelines backend.
 
+
+## Bazel
+
+There are some [know issues](https://github.com/kubeflow/pipelines/issues/3942) with the latest version of Bazel and the kubeflow pipelines repo, therefore use version 0.24.1 or older. 
```

Comment:
```suggestion
There are some [know issues](https://github.com/kubeflow/pipelines/issues/3942) with the latest version of Bazel and the kubeflow pipelines repo, use version 0.24.0.  
```

Reply:
@Bobgy would this be a better way to state it? Maybe we dont need to mention the issues with later versions just that that 0.24.0 should be used?

Reply:
Yes, I think so. All developers and test infra is using this version, I think we should just tell people to use it.